apps/src/javalab/JavalabView.jsx

```diff
@@ -151,18 +183,20 @@ class JavalabView extends React.Component {
                 </button>
               </div>
               <div style={style.buttons}>
-                <button
-                  type="button"
-                  style={style.singleButton}
-                  onClick={this.compile}
+                <JavalabSettings
+                  style={{
```

Comment:
can we extract the button styling out into a variable/function so we don't repeat the code for each button?